content/summary.md

```diff
@@ -29,7 +26,8 @@
 **README.md or README.rst?**
 
 - Functionality-wise both Markdown and RST are very similar.
-- With RST it is a one-liner to generate a table of contents automatically.
+- With RST it is a one-liner to generate a table of contents (TOC) automatically.
+  But nowadays also GitHub generates a TOC for README.md.
```

Comment:
Should the commands for generating the toc for both cases be added here? Or link to where to find them?

Reply:
Great catch. I will add this to this PR.